gms/impl/src/main/resources/index/usage-event/aws_es_index_template.json

```diff
@@ -1,5 +1,5 @@
 {
-  "index_patterns": ["datahub_usage_event-*"],
+  "index_patterns": ["PREFIXdatahub_usage_event-*"],
```

Comment:
PREFIX will get correctly populated here?

Reply:
sed is used to do string replace. Couldn't think of a better way here. Any ideas?

Reply:
I see... hmm not really